addons/im_livechat/static/src/embed/common/chat_bubble_patch.js

```diff
@@ -0,0 +1,12 @@
+import { ChatBubble } from "@mail/core/common/chat_bubble";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(ChatBubble.prototype, {
+    get showImStatus() {
+        return (
+            super.showImStatus &&
+            !(this.thread.correspondent.livechat_member_type === "bot" && this.env.embedLivechat)
```

Comment:
thread in chat bubble doesn't necessarily have a correspondent, e.g. when only self in conversation. Should optional guard it.

Reply:
I would `?.` again here, because we could change code in original code without changing the patch and it may crash because of lack of optional chaining.

The code changes frequently, it happens frequently for original code to be changed but patches are changed only if there's obvious crash or someone finds the problem. Visually `thread.correspondent` doesn't seem to be properly guarded over there ;)